release-cloudbuild.yaml

```diff
@@ -89,6 +89,71 @@ steps:
     secretEnv:
       - USERNAME
 
+  # build client quickstart gcp image - python 3.11
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - '-c'
+      - |
+        docker build \
+          --build-arg ZENML_VERSION=$TAG_NAME \
+          --build-arg PYTHON_VERSION=3.11 \
+          --build-arg CLOUD_PROVIDER=gcp \
+          --target client \
+          -f docker/zenml-quickstart.Dockerfile . \
+          -t $$USERNAME/zenml-public-pipelines:quickstart-$TAG_NAME-py3.11-gcp
+
+        # use latest tags only for official releases
+        if [[ $TAG_NAME =~ ^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)$ ]]; then
+          docker tag $$USERNAME/zenml-public-pipelines:quickstart-$TAG_NAME-py3.11 $$USERNAME/zenml-public-pipelines:quickstart-latest
+        fi         
+
+    id: build-quickstart-3.11-gcp
+    waitFor: [ '-' ]
+    entrypoint: bash
+    secretEnv:
+      - USERNAME
+
+  # build client quickstart aws image - python 3.11
+  - name: gcr.io/cloud-builders/docker
+    args:
+      - '-c'
+      - |
+        docker build \
+          --build-arg ZENML_VERSION=$TAG_NAME \
+          --build-arg PYTHON_VERSION=3.11 \
+          --build-arg CLOUD_PROVIDER=aws \
+          --target client \
+          -f docker/zenml-quickstart.Dockerfile . \
+          -t $$USERNAME/zenml-public-pipelines:quickstart-$TAG_NAME-py3.11-aws
+        
+        # Push to aws - do we have credentials here???
+        docker tag $$USERNAME/zenml:$TAG_NAME-py3.11 339712793861.dkr.ecr.eu-central-1.amazonaws.com/zenml-public-pipelines:quickstart-$TAG_NAME-py3.11-aws
```

Comment:
I doubt you will get creds here, a secret Env would be needed here and in the Azure below. Better check with infra folks for greater details.

Reply:
@AlexejPenner you are pushing it all now to docker registry, right?